src/core/TrainPart.hpp

```diff
@@ -53,24 +54,25 @@ namespace godot {
             virtual void _register_commands();
             virtual void _unregister_commands();
 
-            TMoverParameters *get_mover();
+            [[nodiscard]] TMoverParameters *get_mover() const;
```

Comment:
???

Reply:
https://stackoverflow.com/questions/76489630/explanation-of-nodiscard-in-c17

Reply:
I do not understand. How about other methods? They have return values, too, but without `[[nodiscard]]`.
![image](https://github.com/user-attachments/assets/cc38a794-267b-41b4-8a5d-908d77142178)

`get_mover()` is not changing anything. I do not understand.